### PR TITLE
Badge Colors

### PR DESCRIPTION
**Pull request by Velin92**

### CODE REVIEW

**Review by stefanceriu — APPROVED** [review]
👌

File: Sources/Compound/Colors/CompoundColors.swift [hunk 1]
```diff
@@ -160,6 +160,15 @@ public struct CompoundColors {
     /// This token is a placeholder and hasn't been finalised.
     @available(iOS, deprecated: 17.0, message: "This token should be generated by now.")
     public let _textOwnPill = compound.colorGreen1100
+    /// This token is a placeholder and hasn't been finalised.
+    @available(iOS, deprecated: 17.0, message: "This token should be generated by now.")
+    public let _bgBadgeHighlighted = compound.colorGreen300.opacity(0.11)
```
Comment (pixlwave) [inline Sources/Compound/Colors/CompoundColors.swift:165]: Should be this
```suggestion
    public let _bgBadgeHighlighted = compound.colorAlphaGreen300
```

**Review by pixlwave — COMMENTED** [review]
Can we use `Success` instead of `Highlighted`. There's a system of `success` for green, `info` for blue and `alert` for red that we should follow here.

**Comment by Velin92** [comment]
> Can we use `Success` instead of `Highlighted`. There's a system of `success` for green, `info` for blue and `alert` for red that we should follow here.

But not really a success is more like an highlighted or a positive connotation than a success thing which implies an action

**Pull request merged**